Digital resources for posters presented at conferences.

# Currently presenting: Plant Health 2023

## Abstract submitted

**Pangenome of global _Phytophthora ramorum_ lineages reveals loss of crinkler effector genes and intra-specific plasticity**

Sudden oak death in North America and Sudden larch death in Europe are devastating diseases caused by _Phytophthora ramorum_. The recent discovery that _P. ramorum_ is likely native to Asia has provided further opportunity to investigate its evolutionary history. We used newly assembled, high-quality genome sequences of three Asian and three introduced US lineages of _P. ramorum_ to carry out a pangenome analysis. We focused on comparing putative conserved effector genes. Crinkler genes are an ancient class of oomycete cytoplasmic effectors involved in host necrosis and expanded in most _Phytophthora_ species. Apparently, crinkler genes underwent a recent contraction, with the loss of multiple crinkler families, in _Phytophthora_ sub-clade 8c that contains _P. ramorum_. Furthermore, in the ancestor of both _P. ramorum_ and its sibling species _P. lateralis_ we detected a significant absence of paralogs in the largest core crinkler family. Secreted carbohydrate-active enzymes, believed to influence host range of fungal and oomycete plant pathogens, are just as variable in abundance within _P. ramorum_ as among other clade 8c species. Although the relative abundance of certain effectors was variable among lineages _P. ramorum_ lineages, there was little difference in presence or absence of entire families. This suggests that rather than effector novelty, it is possible that differences in dosage of certain effectors underlies variation in disease severity among lineages on diverse hosts. This work provides novel insights into the history of effector gene evolution.

## Authors

Nicholas C. Cauldron$^1$, Alexandra J. Weisberg$^1$, Caroline M. Press$^2$, Marília Horta Jung$^3$, Tamara Corcobado$^3$, Joan F. Webber$^4$, Koji Kageyama$^5$, Ayaka Hieno$^5$, Hayato Masuya$^6$, Seiji Uematsu$^7$, Bruno Scanu$^8$, Clive M. Brasier$^4$, Thomas Jung$^3$, Jeff H. Chang$^1$, and Niklaus J. Grünwald$^2$

1 Department of Botany and Plant Pathology, Oregon State University, Corvallis, Oregon 97331, USA
2 Horticultural Crops Disease and Pest Management Research Unit, USDA Agricultural Research Service, Corvallis, OR 97330, USA
3 Phytophthora Research Centre, Faculty of Forestry and Wood Technology, Mendel University in Brno, 613 00 Brno, Czech Republic
4 Forest Research, Alice Holt Lodge, Farnham, Surrey, GU10 4LH, UK
5 River Basin Research Center, Gifu University, Gifu 501-1193, Japan
6 Forestry and Forest Products Research Institute (FFPRI), Tsukuba, Ibaraki 305-8687, Japan
7 Department of Bioregulation and Bio-interaction, Tokyo University of Agriculture and Technology, Fuchu, Tokyo 183-8509, Japan
8 Department of Agricultural Sciences, University of Sassari, 07100 Sassari, Italy

## Extended methods

•Sequenced one strain each of clonal lineages epidemic in US (NA1, NA2, EU1), three from candidate center of origin (Indochina/Vietnam: IC1, IC2; Nippon/Japan: NP1)

•Assembled NA1, EU1, IC1, IC2, NP1 from PacBio data and NA2 from Nanopore

•Scaffolded chromosomes with intra- and interspecific homology, curated suspicious joins

•Gene annotation: BRAKER (Hoff et al. 2019), supplemented RXLR and crinkler effector candidates from all ORFs. RXLRs and crinklers annotated using custom pipelines with HMM and string searches of known motifs.

•Glycoside hydrolase domains: dbCAN2 (Lombard et al. 2014)

•Synteny: GENESPACE (Lovell et al. 2022) 

•Phylogenetics: Funannotate (Palmer and Stajich 2020) - IQ-Tree on trimmed alignment of 118 single-copy BUSCO proteins (Nguyen et al. 2015, Seppey et al. 2019)

•Orthogroups: OrthoFinder (Emms), visualized with UpSetR (Gehlenborg 2016)

•Gene family birth-death: CAFÉ 5 (Mendes et al. 2020)

•Gene annotation: BRAKER (Hoff et al. 2019), supplemented RXLR and crinkler effector candidates from all ORFs

## Funding

This work was supported by grants from USDA ARS (2072-22000-041-000-D), USDA NIFA  
(2023-67013-39918) and USDA APHIS to N.J.G. and a USDA NIFA predoctoral fellowship 2021-67034-35171 to NC.

## References

1.     McDonald, Bruce A., and Celeste Linde. “Pathogen Population Genetics, Evolutionary Potential, and Durable Resistance.” _Annual Review of Phytopathology_ 40, no. 1 (2002): 349–79. [https://doi.org/10.1146/annurev.phyto.40.120501.101443](https://doi.org/10.1146/annurev.phyto.40.120501.101443).

2.     Hessenauer, Pauline, Nicolas Feau, Upinder Gill, Benjamin Schwessinger, Gurcharn S. Brar, and Richard C. Hamelin. “Evolution and Adaptation of Forest and Crop Pathogens in the Anthropocene.” _Phytopathology®_ 111, no. 1 (January 2021): 49–67. [https://doi.org/10.1094/PHYTO-08-20-0358-FI](https://doi.org/10.1094/PHYTO-08-20-0358-FI).

3.     Kronmiller, Brent A., Nicolas Feau, Danyu Shen, Javier F. Tabima, Shahin S. Ali, Andrew D. Armitage, Felipe Arredondo, et al. “Comparative Genomic Analysis of 31 _Phytophthora_ Genomes Reveals Genome Plasticity and Horizontal Gene Transfer.” _Molecular Plant-Microbe Interactions®_, October 28, 2022, MPMI-06-22-0133-R. [https://doi.org/10.1094/MPMI-06-22-0133-R](https://doi.org/10.1094/MPMI-06-22-0133-R).

4.     LeBoldus, Jared M., Sarah M. Navarro, Norma Kline, Gabriela Ritokova, and Niklaus J. Grünwald. “Repeated Emergence of Sudden Oak Death in Oregon: Chronology, Impact, and Management.” _Plant Disease_, April 29, 2022, PDIS-02-22-0294-FE. [https://doi.org/10.1094/PDIS-02-22-0294-FE](https://doi.org/10.1094/PDIS-02-22-0294-FE).

5.     Jung, Thomas, Bruno Scanu, Clive M. Brasier, Joan Webber, Ivan Milenković, Tamara Corcobado, Michal Tomšovský, et al. “A Survey in Natural Forest Ecosystems of Vietnam Reveals High Diversity of Both New and Described _Phytophthora_ Taxa Including _P. Ramorum_.” _Forests_ 11, no. 1 (January 2020): 93. [https://doi.org/10.3390/f11010093](https://doi.org/10.3390/f11010093).

6.     Hoff, Katharina, Alexandre Lomsadze, Mark Borodovsky, and Mario Stanke. “Whole-Genome Annotation with BRAKER.” _Methods in Molecular Biology (Clifton, N.J.)_ 1962 (2019): 65–95. [https://doi.org/10.1007/978-1-4939-9173-0_5](https://doi.org/10.1007/978-1-4939-9173-0_5).

7.     Lovell, John T, Avinash Sreedasyam, M Eric Schranz, Melissa Wilson, Joseph W Carlson, Alex Harkess, David Emms, David M Goodstein, and Jeremy Schmutz. “GENESPACE Tracks Regions of Interest and Gene Copy Number Variation across Multiple Genomes.” Edited by Detlef Weigel. _ELife_ 11 (September 9, 2022): e78526. [https://doi.org/10.7554/eLife.78526](https://doi.org/10.7554/eLife.78526).

8.     Emms, David M., and Steven Kelly. “OrthoFinder: Phylogenetic Orthology Inference for Comparative Genomics.” _Genome Biology_ 20, no. 1 (November 14, 2019): 238. [https://doi.org/10.1186/s13059-019-1832-y](https://doi.org/10.1186/s13059-019-1832-y).

9.     Mendes, Fábio K, Dan Vanderpool, Ben Fulton, and Matthew W Hahn. “CAFE 5 Models Variation in Evolutionary Rates among Gene Families.” _Bioinformatics_ 36, no. 22–23 (December 1, 2020): 5516–18. [https://doi.org/10.1093/bioinformatics/btaa1022](https://doi.org/10.1093/bioinformatics/btaa1022).

10.  Kamoun, Sophien, Michael C. Zody, Rays H. Y. Jiang, Robert E. Handsaker, Liliana M. Cano, Manfred Grabherr, Chinnappa D. Kodira, et al. “Genome Sequence and Analysis of the Irish Potato Famine Pathogen _Phytophthora Infestans_.” _Nature_ Volume 461, no. Number 7262 (September 17, 2009): 393–98.

11.  Fang, Yufeng, Marco A. Coelho, Haidong Shu, Klaas Schotanus, Bhagya C. Thimmappa, Vikas Yadav, Han Chen, et al. “Long Transposon-Rich Centromeres in an Oomycete Reveal Divergence of Centromere Features in Stramenopila-Alveolata-Rhizaria Lineages.” _PLOS Genetics_ 16, no. 3 (March 9, 2020): e1008646. [https://doi.org/10.1371/journal.pgen.1008646](https://doi.org/10.1371/journal.pgen.1008646).

12.  Rafiei, Vahideh, Heriberto Vélëz, and Georgios Tzelepis. “The Role of Glycoside Hydrolases in Phytopathogenic Fungi and Oomycetes Virulence.” _International Journal of Molecular Sciences_ 22, no. 17 (August 28, 2021): 9359. [https://doi.org/10.3390/ijms22179359](https://doi.org/10.3390/ijms22179359).

13.  Dong, Suomeng, and Wenbo Ma. “How to Win a Tug-of-War: The Adaptive Evolution of _Phytophthora_ Effectors.” _Current Opinion in Plant Biology_ 62 (August 1, 2021): 102027. [https://doi.org/10.1016/j.pbi.2021.102027](https://doi.org/10.1016/j.pbi.2021.102027).